frontend/app/src/main/java/com/apptive/japkor/ui/requiredinfo/steps/Step3Content.kt

```diff
@@ -37,11 +41,35 @@ fun Step3Content() {
         Spacer(modifier = Modifier.height(50.dp))
 
         // 프로필 사진 등록 UI 구현 (예시)
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(Color.White, shape = RoundedCornerShape(16.dp)),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy(12.dp)
+        ) {
+            CustomOutlinedTextField(
+                value = "",
+                onValueChange = {},
+                placeholder = "학력(인증)"
+            )
+            CustomOutlinedTextField(
+                value = "",
+                onValueChange = {},
+                placeholder = "재산(인증)"
+            )
+            CustomOutlinedTextField(
+                value = "",
+                onValueChange = {},
+                placeholder = "기타정보"
+            )
+
+        }
+
         CustomText(
-            text = "Step 3: 화면",
+            text = " '학력' 및 '재산' 정보는 선택 입력 사항입니다.\n 입력시, 더 빠르고 정확한 매칭에 도움을 줄 수 있습니다.",
```

Comment:
```suggestion
            text = "'학력' 및 '재산' 정보는 선택 입력 사항입니다.\n입력시, 더 빠르고 정확한 매칭에 도움을 줄 수 있습니다.",
```
이렇게 수정 부탁드려요~
줄마다 맨앞에 공백 들어갔네요~